plugins/services/src/js/constants/ServiceStatus.ts

```diff
@@ -163,10 +163,7 @@ function showProgressBar(status: Status): boolean {
   return [DEPLOYING, WAITING, DELAYED, RECOVERING, DELETING].includes(status);
 }
 
-function toIcon(status: Status | null): StatusIcon | null {
-  if (status === null) {
```

Comment:
@TattdCodeMonkey i removed that safeguard here, as the types say that we should be good. if you had a crash because of this, please reintroduce it!